microsoft/testsuites/display/drm.py

```diff
@@ -67,6 +68,41 @@ def verify_dri_node(self, node: Node, log: Logger) -> None:
             "dri node not populated for hyperv_drm"
         ).matches("hyperv_drm")
 
+    @TestCaseMetadata(
+        description="""
+        This case is to check this patch
+        https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=19b5e6659eaf537ebeac90ae30c7df0296fe5ab9   # noqa: E501
+
+        Step,
+        1. Get dmesg output.
+        2. Check no 'Unable to send packet via vmbus' shown up in dmesg output.
+        """,
+        priority=2,
+    )
+    def verify_no_error_output(self, node: Node, log: Logger) -> None:
+        dmesg = node.tools[Dmesg]
+        assert_that(dmesg.get_output()).described_as(
```

Comment:
No need to use a variable for a lookup that is used once

```suggestion
        assert_that(node.tools[Dmesg].get_output()).described_as(
```